mobify/progressive-web-scaffold
# Cart estimate shipping

This PR replaces the hardcoded shipping rate with correct values, and implements the estimate shipping modal submission

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1110
 **Linked PRs**: n/a

## Changes
- Refactored estimate shipping API call to be re-used on cart & checkout shipping pages
- Fetched shipping method & process checkout data when cart data is received
- Wrapped estimate shipping form in redux-form so we can access selected values
- Added action to submit estimate shipping form 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart
- click "calculate" on the cart page
- enter values in the estimate shipping form and submit
- a network request should go out with the values you entered (note all shipping is the same regardless of location right now, so shipping rate will not change)
- proceed to the checkout shipping page
- enter values for country, region and postal code
- confirm that a network request goes out with the new values entered (again shipping rate will not change)
